protocol/messenger_handler.go

```diff
@@ -738,15 +777,14 @@ func (m *Messenger) HandleContactUpdate(state *ReceivedMessageState, message pro
 		contact.LastUpdated = message.Clock
 		state.ModifiedContacts.Store(contact.ID, true)
 		state.AllContacts.Store(contact.ID, contact)
-		/* Disabling for now in order to avoid duplicated contact requests in activity center
```

Comment:
Perhaps add a `TODO` for removing or reactivating when the right circumstances have been met?

Reply:
OH, I'm so slow! I completely misread the diff. Ignore this 🤦